sample/other-junit.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites name="other tests" tests="1" failures="0" errors="0" time="0.232">
+  <testsuite name="undefined" errors="0" failures="0" skipped="0" timestamp="2021-11-02T13:28:36" time="0.189" tests="1">
+    <testcase classname=" something" name=" works somehow" time="0.001">
+    </testcase>
+  </testsuite>
+</testsuites>
```

Comment:
Is this file being used? Tried searching to see if anything would use it as an input or output but couldn't tell.

Reply:
It's used to test the scenario where we only set the test-results-path (without specifying the framework): https://github.com/CircleCI-Public/node-orb/pull/100/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R247